docker-compose.yaml

```diff
@@ -8,6 +8,8 @@ services:
       - 8000:8000
     volumes: 
       - ./services/backend/api:/app/api
+    env_file:
+      - .env
```

Comment:
This is a simple way to specify that you want docker to load environment vars from a file with a `key=value` format.